### PR TITLE
feat(storage-upload): reads file size limit from service

### PR DESCRIPTION
**Pull request by farinalvaro**
Now on https://github.com/rsksmart/rif-marketplace-ui/pull/662

~- Gets file size limit from upload-service~
~- Shows max size text~
~- Disables upload button until there is content to be uploaded~

~By default it's set to 1GB in UI. This allows us to always handle a value without having to "pause" the flow until we have the max size defined~

### CODE REVIEW

File: src/context/Services/storage/upload/Context.tsx [hunk 1]
```diff
@@ -23,6 +29,10 @@ export type ContextName = typeof contextID
 export const initialState: State = {
   contextID,
   status: {},
+  fileSizeLimit: Big(UNIT_PREFIX_POW2.GIGA), // 1GB by default
+  isLoading: {
+    sizeLimit: false,
+  },
```
Comment (farinalvaro) [inline src/context/Services/storage/upload/Context.tsx:35]: Created an object for this so we can add a flag for uploading files action

**Review by julianmrodri — APPROVED** [review]
LGTM! 🚀

File: src/context/Services/storage/upload/Context.tsx (hunk 1 above)
Comment (farinalvaro) [reply src/context/Services/storage/upload/Context.tsx:35]: Actually I noticed that this should be handled at component level and not in context so ignore my comment 😄

**Comment by farinalvaro** [comment]
Closing in favor of https://github.com/rsksmart/rif-marketplace-ui/pull/662

**Pull request closed without merging**